### ISSUE TITLE
Look over docs and fix typos

### ISSUE THREAD

**Issue by sdegutis** [sdegutis/mjolnir#42]
I found at least one.

**Comment by sdegutis** [comment]
That'd be awesome, thanks.

**Comment by sdegutis** [comment]
Let's use this issue for all bugs in docs, and a separate issue for each suggested rename or question of clarity (i.e. non-bug related doc issues). For example of something that doesn't belong here, see #44.

**Comment by sdegutis** [comment]
It might be easiest to read through [the HTML docs](http://sdegutis.github.io/hydra/api.html), especially after #54 is finished.

**Comment by sdegutis** [comment]
I fixed all the ones I've found so far. But it's really tedious so I won't look through anymore for now.

**Comment by sdegutis** [comment]
Done.
